_field-types/supported-field-types/star-tree.md

```diff
@@ -0,0 +1,148 @@
+
+## Star tree mapping parameters
+Specify star-tree configuration under `config` section. All parameters are final and cannot be modified without reindexing documents.
+
+### Ordered dimensions
+The `ordered_dimensions` are fields based on which the metrics will be aggregated in star-tree index. Star Tree index will be picked for query optimizations only if all the fields in the query are part of the `ordered_dimensions`. This is a required property as part of star-tree configuration.
+- The order of dimensions matter and you can define the dimensions ordered from the highest cardinality to the lowest cardinality for efficient storage and query pruning. 
+- Avoid high cardinality fields as dimensions , because it'll affect storage space, indexing throughput and query performance adversely.
+- Currently, supported fields for `ordered_dimensions` are of [numeric field types](https://opensearch.org/docs/latest/field-types/supported-field-types/numeric/) with the exception of `unsigned_long`. 
+  - Support for other field_types such as `keyword` , `ip` is coming as part of upcoming releases.
+- Minimum of `2` and maximum of `10` dimensions are supported per Star Tree index.
+
+#### Properties
+
+| Parameter            | Required/Optional | Description                                                                                                                                                                                                                                                                                                                                                                                                                         | 
+|:---------------------| :--- |:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name` | Required | Name of the field which should also be present in `properties` as part of index `mapping` and ensure `doc_values` is `enabled` for associated fields.
+
+### Metrics
+Configure fields for which you need to perform aggregations. This is required property as part of star-tree configuration.
+- Currently, supported fields for `metrics` are of [numeric field types](https://opensearch.org/docs/latest/field-types/supported-field-types/numeric/) with the exception of `unsigned_long`.
+- Supported metric aggregations include `Min`, `Max`, `Sum`, `Avg` and `Value_count`. 
+  - `Avg` is a derived metric based on `Sum` and `Value_count` and is not indexed and is derived on query time. Rest are base metrics which are indexed.
+- Upto `100` base metrics are supported per Star Tree index.
```

Comment:
This can be elaborated to make sure end user understands there are different ways of reaching this limit.